src/marqo/tensor_search/add_docs.py

```diff
@@ -192,4 +197,54 @@ def determine_document_dict_field_type(field_name: str, field_content, mappings:
         else:
             raise base_exceptions.InternalError(f"Invalid dict field type: '{mappings[field_name]['type']}' for field: '{field_name}' in mappings. Must be one of {[t.value for t in enums.MappingsObjectType]}")
     else:
-        return None
\ No newline at end of file
+        return None
+    
+
+def determine_text_prefix(request_level_prefix: str, marqo_index: MarqoIndex, prefix_type: str) -> str:
```

Comment:
This shouldn't be in add_docs anymore, as it's used for both chunk prefixes and query prefixes